src/newriff.js

```diff
@@ -0,0 +1,85 @@
+const maxApi = require('max-api');
+const scribble = require('scribbletune');
+const _ = require('lodash');
+
+maxApi.post('hello from v4 in riff');
+
+maxApi.addHandler('makeClip', () => {
+  //we create an event handler function to contain the script. For more information on how this works, see Node for Max documantation https://docs.cycling74.com/nodeformax/api/
+  const constructClip = (async () => {
+    const parameters = await maxApi.getDict('parameters'); //we fetch the main dictionary "parameters"
```

Comment:
Love this way of getting the params from the dict instead of splitting a string :P

Reply:
Well hey, it took me forever to figure this out. The Dict object is pretty neat actually